tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "noEmit": true,
     "skipLibCheck": true,
     "resolveJsonModule": true,
-    "strict": true
+    "strict": true,
+		"types": ["node", "jest", "cypress", "@testing-library/cypress"]
```

Comment:
Should we only add this to the `cypress/` directory?

E.g. https://github.com/DeFiCh/scan/tree/main/cypress

Reply:
Updated.

Also added `declare global` to custom command to remove ts complain